SEC/Equities/EquityInstruments.rdf

```diff
@@ -406,12 +420,73 @@
 		<skos:definition xml:lang="en">position in an equity instrument</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;EquityRedemptionProvision">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;RedemptionProvision"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasMinimumRedemptionPrice"/>
+				<owl:onClass rdf:resource="&fibo-fbc-fi-ip;MarketPrice"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasRedemptionPremium"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
```

Comment:
Not a percentage?

Reply:
One can derive a percentage from an amount, or vice versa, but it could be declared either way.  Most of the definitions we found were for an amount over par rather than a percentage.  In either case you would need to know the par value in order to calculate a percentage, which is not necessarily required if you have a straight amount.  Redemption provisions typically applies to preferred or restricted shares, not to a  common share.

Reply:
OK